### PR TITLE
update readme installing obelisk with wget.

### PR DESCRIPTION
**Pull request by kayhide**
At the Lambda Jam workshop, I had a trouble of installing and found a way.
This may help somebody, I am sharing what I did.

I am on macOS with nix package manager installed.

```console
$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.14.4
BuildVersion:	18E226
$ nix --version
nix (Nix) 2.2
```

### CODE REVIEW

**Comment by ali-abrar** [comment]
Thanks @kayhide - do you happen to have the error that you encountered? It would be great if you didn't get that error in the first place!

File: README.md [hunk 1]
```diff
@@ -44,6 +44,14 @@ Obelisk provides an easy way to develop and deploy your [Reflex](https://github.
           sudo launchctl start org.nixos.nix-daemon
           ```
 1. Install obelisk: `nix-env -f https://github.com/obsidiansystems/obelisk/archive/master.tar.gz -iA command`
+    * In case of failing this command due to a ssh error, the following may work:
+        ```
+        $ wgat https://github.com/obsidiansystems/obelisk/archive/master.tar.gz
```
Comment (ali-abrar) [inline README.md]: wgat -> wget

**Comment by kayhide** [comment]
Hi @ali-abrar , thanks for your response.

I currently don't have the error because at that workshop, I was working on the company's mac book, which is now at the office.
I will try it and see if it reproduces at the office next week.

File: README.md (hunk 1 above)
Comment (kayhide) [reply README.md]: Fixed!

**Comment by kayhide** [comment]
I does not reproduce now.
Since the venue had a poor internet access, the cause might have been related to the network issue somehow.

**Comment by 3noch** [comment]
Let us know if you run into anything like this again!

**Pull request closed without merging**